### ISSUE TITLE
phpseclib conflict with other packages

### ISSUE THREAD

**Issue by Fossil01** [tymondesigns/jwt-auth#1130]
I'm trying to install https://packagist.org/packages/laravelcollective/remote version 5.4.0 but it fails with the error below. AFAIK jwt-auth uses phpseclib 0.3.10.

How do I fix this, I'm on Laravel 5.4? I tried updating jwt-auth to 1.0.0-beta.3 by changing composer.json but that threw a crapton of errors so I switched back.

Is there some update guide from 0.5 to 1.0 beta? Or someone here who can tell me what to do?

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for laravelcollective/remote ^5.4.0 -> satisfiable by laravelcollective/remote[v5.4.0].
    - Conclusion: remove phpseclib/phpseclib 0.3.10
    - Conclusion: don't install phpseclib/phpseclib 0.3.10
    - laravelcollective/remote v5.4.0 requires phpseclib/phpseclib ^2.0 -> satisfiable by phpseclib/phpseclib[2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4].
    - Can only install one of: phpseclib/phpseclib[2.0.0, 0.3.10].
    - Can only install one of: phpseclib/phpseclib[2.0.1, 0.3.10].
    - Can only install one of: phpseclib/phpseclib[2.0.2, 0.3.10].
    - Can only install one of: phpseclib/phpseclib[2.0.3, 0.3.10].
    - Can only install one of: phpseclib/phpseclib[2.0.4, 0.3.10].
    - Installation request for phpseclib/phpseclib (locked at 0.3.10) -> satisfiable by phpseclib/phpseclib[0.3.10].

Installation failed, reverting ./composer.json to its original content.
```

**Comment by fredroo** [comment]
same problem, for me use require in composer.json with:

"tymondesigns/jwt-auth": "1.0.0-beta.3"

**Comment by bahman2216** [comment]
Same problem.
I just install **"namshi/jose": "^7.2"** But have a new problem:
`- tymon/jwt-auth 0.5.11 requires namshi/jose 5.0.* -> satisfiable by namshi/jose[5.0.0, 5.0.1, 5.0.2] but these conflict with your requirements or minimum-stability.
`

**Comment by tymondesigns** [comment]
fixed via `0.5.12`
